pkg/apis/scorecard/v1alpha3/configuration_types.go

```diff
@@ -22,6 +22,9 @@ type Configuration struct {
 
 	// Storage is the optional storage configuration
 	Storage Storage `json:"storage,omitempty" yaml:"storage,omitempty"`
+
+	// ServiceAccount is the optional way to set a non-default SA
```

Comment:
Nit: maybe a little more detail here? Try to describe what it is/does as a primary concern, and then if necessary go into the "why" as another GoDoc paragraph. Something along the lines of the following is probably sufficient:
```suggestion
	// ServiceAccount is the service account under which scorecard tests are run. This field is optional. If left unset, the `default` service account will be used.
```